src/transformers/training_args.py

```diff
@@ -1837,9 +1837,9 @@ def _setup_devices(self) -> "torch.device":
         requires_backends(self, ["torch"])
         logger.info("PyTorch: setting up devices")
         if not is_sagemaker_mp_enabled():
-            if not is_accelerate_available(min_version="0.20.1"):
+            if not is_accelerate_available():
                 raise ImportError(
-                    "Using the `Trainer` with `PyTorch` requires `accelerate>=0.20.1`: Please run `pip install transformers[torch]` or `pip install accelerate -U`"
+                    "Using the `Trainer` with `PyTorch` requires `accelerate>=0.21.0`: Please run `pip install transformers[torch]` or `pip install accelerate -U`"
```

Comment:
Can we future proof this by instead importing `ACCELERATE_MIN_VERSION` from `transformers.utils.import_utils` and have that in the f-string? Thanks!

Reply:
Yes, that was my first thought. However, `ACCELERATE_MIN_VERSION` is not re-exported.  Should I really import `transformers.utils.import_utils`?

Reply:
You can add it to `transformers.utils.__init__.py` and add it that way (there's already a few const's in that section of the imports)

Reply:
Okay, done. Please resolve the comment if correct.

Reply:
Nice! Yes indeed